client/src/main.rs

```diff
@@ -316,6 +316,30 @@ fn init_wayland() -> (WaylandState, GlobalManager) {
             ));
         }
     });
+    globals.instantiate_exact(
```

Comment:
This is getting repeaty, you should add a macro or something to clean this up (or can we do it with a function?)